website/docs/docs/deploy/ci-jobs.md

```diff
@@ -14,10 +14,21 @@ You can set up Slim [continuous integration](/docs/deploy/continuous-integration
     - If you’re using GitLab, you must use a paid or self-hosted account which includes support for GitLab webhooks.
     - If you previously configured your dbt project by providing a generic git URL that clones using SSH, you must reconfigure the project to connect through dbt Cloud's native integration.
 
-## Set up Slim CI jobs
+## Set up CI jobs
 
 dbt Labs recommends that you create your Slim CI job in a dedicated dbt Cloud [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) that's connected to a staging database. Having a separate environment dedicated for CI will provide better isolation between your temporary CI schemas builds and your production data builds. Additionally, sometimes teams need their Slim CI jobs to be triggered when a PR is made to a branch other than main. If your team maintains a staging branch in your release process, having a separate environment will allow you to set a [custom branch](/faqs/environments/custom-branch-settings), and accordingly the CI job in that dedicated environment will be triggered only when PRs are made to the specified, custom branch.
 
+:::tip Join our beta 
+
+dbt Labs is currently running a beta that provides improved UI updates for setting up CI jobs. For docs on this, click on the **Beta version** tab below.
+
+If you're interested in joining our beta, please [contact us](mailto:support@getdbt.com).
+
```

Comment:
is the wording ok for the beta callout?

Reply:
yep!

Reply:
@nghi-ly - actually, let's point to this [form](https://docs.google.com/forms/d/e/1FAIpQLSd2Szy3QYrDPSpJiqqhgl7xa6eo1VQoLm35Xg81R64IefBgoQ/viewform) instead